ts/Extensions/SeriesLabel/SeriesLabel.ts

```diff
@@ -48,7 +48,7 @@ import type SymbolOptions from '../../Core/Renderer/SVG/SymbolOptions';
 import A from '../../Core/Animation/AnimationUtilities.js';
 const { animObject } = A;
 import Chart from '../../Core/Chart/Chart.js';
-import FU from '../../Core/FormatUtilities.js';
+import FU from '../../Core/Templating.js';
```

Comment:
Not a blocker: I think it would be better to either use `F` or `Templating` everywhere. FU is just.. misleading ;)